docs/source/demos/finnegans_wake_demo.rst

```diff
@@ -0,0 +1,55 @@
+Finnegan's Wake Demo
+====================
+
+.. figure:: https://cdn-images-1.medium.com/max/600/0*b42NPOnflrY07rEf.jpg
+    :width: 100%
+
+Overview
+--------
+
+.. important::
+
+    This demo requires connecting to a running network. By default the demo is hardcoded to connect to the local demo fleet.
+
+
+This demo is an example of a NuCypher decentralized network allowing Alice to share
+data with Bob using proxy re-encryption. This enables the private sharing of data between
+participants in public consensus networks, without revealing data keys to intermediary entities.
```

Comment:
This sentence is said repeatedly by us (perhaps from the whitepaper, I don't remember), but I'm starting to think we shouldn't say "_participants in public consensus networks_". It's too limited. Let's recall that Bob doesn't need a priori to be a participant of a consensus network to be granted access.